module/frame_parser.py

```diff
@@ -0,0 +1,12 @@
+import re
```

Comment:
this file should be removed as it has [already been implemented in python build](https://github.com/R2D2-2019/r2d2-python-build/pull/31)